packages/cli/main.js

```diff
@@ -2,23 +2,44 @@
 
 import hrp from "http-request-plus";
 import { format, parse } from "json-rpc-protocol";
+const WebSocket = require("ws");
+const util = require("util");
 
 async function main() {
   const params = require("minimist")(process.argv.slice(2));
   const method = params._[0];
 
-  const response = parse(
-    await hrp
-      .post("http://localhost:4000/api/", {
-        body: format.request(0, method, params),
-      })
-      .readAll("utf-8")
-  );
+  if (method === "displayEvents") {
```

Comment:
`displayEvents` is not an api method. IMHO, we should use this syntax `cli --watch` to watch events.

Reply:
This could help you: https://github.com/JorgeBucaran/getopts#optsstopearly